HW02_B_Mimic_starter.md

```diff
@@ -0,0 +1,81 @@
+HW02\_B\_Graph-Mimic
+================
+Julia Shangguan
+
+``` r
+library("ggplot2")
+library("magrittr") 
+data("diamonds")
+data("mpg")
+data("iris")
+theme_set(theme_bw()) 
+library("ggrepel")
+```
+
+## HW02 Part B
+
+### Graph 1
+
+``` r
+ggplot(diamonds, aes(x = cut, fill = clarity)) +
+  geom_bar(position = "dodge") + 
+  annotate("rect", xmin = 4.5, xmax = 5.5, ymin = 0, ymax = 5000,fill = "black", alpha = 0.2) +
+  annotate("text", x = 4, y = 4500, label = "My Best Diamonds,\nof course") +
```

Comment:
This is what I was looking for! And it uses exactly the type of code I was expecting. I think I was trying to use just geom_rect() or something similar, and the rectangle that was created actually took up space on the graph (like when you are trying to add an image to Microsoft Word and it makes the formatting all funky.